src/components/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+const Header = () => {
+    return (
+      <header className="Header">
```

Comment:
Good use of a semantic html element - header is the most appropriate one to use here, well done!